app/views/events/edit.html.erb

```diff
@@ -0,0 +1,36 @@
+<%= link_to("<< Back to List", events_path, :class => 'back-link') %>
+
+<div class="event edit">
+    <h2>Edit Event</h2>
+    <%= form_for(@event) do |f| %>
+        <table summary="Event form fields">
+            <tr>
+                <th>Name</th>
+                <td><%= f.text_field(:name) %></td>
+            </tr>
+            <tr>
+                <th>Date</th>
+                <td><%= f.date_select(:date) %></td>
+            </tr>
+            <tr>
+                <th>Time</th>
+                <td><%= f.time_select(:time) %></td>
+            </tr>
+            <tr>
+                <th>Type</th>
+                <td><%= f.text_field(:event_type) %></td>
+            </tr>
+            <tr>
+                <th>Attendance Points</th>
+                <td><%= f.text_field(:event_type) %></td>
```

Comment:
I would make this a number field with a step of like `0.0001` so that it is verified as a number but it can be any real number.